Feature/seext 10469 extend drop downs nav bar style name to have background color of navbar

tomislav-arambasic opened the pull request:
Adjusted the background color & text color of dropdown for `horizontal` style.
`Prime` was the only theme that had "weird" colors for dropdown.

Default dropdown color values for each theme:
![Prime](https://user-images.githubusercontent.com/57353746/134372935-e102ff72-2347-420f-a5a8-4ebc2b0dd802.png)
![Rose](https://user-images.githubusercontent.com/57353746/134372939-7c670877-00cd-4ec9-9b37-2a1134115f87.png)
![Bleu](https://user-images.githubusercontent.com/57353746/134372924-021e64d9-bd32-492f-aea5-205e4ae49862.png)
![Noir](https://user-images.githubusercontent.com/57353746/134372929-0cda57cb-7bde-417a-a48b-40bac1ab0680.png)


Prime, after adjusting new dropdown theme properties:
![Prime after](https://user-images.githubusercontent.com/57353746/134373002-a0d02115-cdef-494c-abac-8482bb950791.png)

majaklajic reviewed (approved):
It looks good to me, but be sure to test all layout combinations...

sstimac reviewed (changes requested):
Discussed in person with Tomislav. We need to make some other changes here

tomislav-arambasic commented:
@sstimac @toroz 

Changed per request.
I've updated PR description & added screenshots showing default dropdown UI per theme & dropdown UI for Prime theme, after changing new dropdown theme theme properties - bg & text colors.

tomislav-arambasic commented:
Closing this, as I've opened a new PR (had to change name of the task).
New PR -> https://github.com/shoutem/ui/pull/641
